### PR TITLE
Add Alpine support

### LINKED ISSUE #55: Alpine support
Just a quick question, rather than a trouble ticket. Are there any plans to support Alpine distros in the future? The biggest difference has to do with installing prerequisites - it uses `apk add` instead of `apt-get install`, basically.

### PR DESCRIPTION
**Pull request by mlocati**
Fix #55

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
-  - master
+    - master
 
 services:
   - docker
 
-env:
-  global:
-    - secure: "vsl8z41oBky/MDRDKqs74Bif6BJldCPFVdl2lbPcjehOYG5cZ4YRLIEuhgJfnOglCGdyY6gh/2umSowRI4r5mC/ZR42RKJzWj9xQX2usJbzeOeZj+5eQv5im1DVwRmigUMzQOS743bd/zEeR9J3MLMgTGhKWEPKOJI6CHseUIKazAc0Z8vH6055YtkhGMqnolOe9h0gJx/sLP18qo2LTw82SyjOAMfbyYuHpiLznAqIOpNtwyj9WAujE3qN2oCo+9ALOnnrlINPqeLjnIOxRcSgYO4wIGPGSXIDTquuUht0McszOLuC9/kc6RybcyxmcqsssYRK2y0DQuavzyja5UaB4Pzf2vMrlbozAxdtF3oadZzfsAYfbhzaLbx0g1aAzL1CTJvXrWkZQgiy2c7tboXKJ3SvOqDp+GXEw08kCTpUq1AYSLw8ExlqcWZGiQxJfnfA05fWOkf8xM38ZQ+LPUHyGrt+PIw8FPAGhOV4E+BmHUlOoXOwoNllCEJ4QNkwxYT90eAMS5Rw8q6KASedwDFwyPCv8VtpNEtsl/FVa3m8/6wA7QmtuPxg7E2wceSkzsqHEcGLzERb4DnhVfrW0wv8zRw92vsnBmZdxusBT5uuyp9h5asjj6YhhVJ52g2NdCAGJlobiPk5X4uifshcdlcS8vF/jBMpI03XyJU4V4xc="
+text-extensions-job-template: &text-extensions-job-template
```
Comment (b1rdex) [inline .travis.yml]:
```suggestion
test-extensions-job-template: &test-extensions-job-template
```

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
+  stage: Test extensions
+  if: type = pull_request
+  script: ./scripts/travisci-test-extensions
 
-install:
-  - true
+jobs:
+  include:
+    - <<: *text-extensions-job-template
```
Comment (b1rdex) [inline .travis.yml]:
```suggestion
    - <<: *test-extensions-job-template
```

File: .travis.yml [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
+      name: Test extensions on Alpine Linux
+      env: DOCKER_DISTRO=alpine
 
-script:
-  - if test "${TRAVIS_EVENT_TYPE:-}" = 'pull_request'; then ./scripts/travisci-test-new-extensions; fi
-  - if test "${TRAVIS_EVENT_TYPE:-}" = 'push' && test "${TRAVIS_BRANCH:-}" = 'master'; then ./scripts/travisci-update-readme; fi
+    - <<: *text-extensions-job-template
```
Comment (b1rdex) [inline .travis.yml]:
```suggestion
    - <<: *test-extensions-job-template
```

File: README.md [hunk 4]
```diff
@@ -157,3 +154,8 @@ Some extension has special requirements:
   
   Test: gd, zip
   ```
+
+
+## Do you want to really say thank you?
+
+You can offer me a [montly coffee](https://github.com/sponsors/mlocati) or a [one-time coffee](https://paypal.me/mlocati) :wink:
```
Comment (b1rdex) [inline README.md]:
```suggestion
You can offer me a [monthly coffee](https://github.com/sponsors/mlocati) or a [one-time coffee](https://paypal.me/mlocati) :wink:
```

File: .travis.yml (hunk 1 above)
Comment (mlocati) [reply .travis.yml]: :+1: Thanks!

**Comment by mlocati** [comment]
I tested all the extensions locally a d everything seems to be ok.

**Pull request merged**